pkg/api/common/const.go

```diff
@@ -53,120 +57,48 @@ const (
 )
 
 const (
-	// KubernetesVersion1Dot10Dot0RC1 is the string for kubernetes 1.10.0-rc.1
```

Comment:
These consts were not adding to readability/maintainability, and runtime optimization is not a concern